aggregator/flush_mgr.go

```diff
@@ -303,3 +315,15 @@ func newBucket(interval time.Duration, scope tally.Scope) *flushBucket {
 func (b *flushBucket) Add(flusher PeriodicFlusher) {
 	b.flushers = append(b.flushers, flusher)
 }
+
+func (b *flushBucket) Remove(flusher PeriodicFlusher) error {
+	numFlushers := len(b.flushers)
+	for i := 0; i < numFlushers; i++ {
+		if b.flushers[i] == flusher {
+			b.flushers[i], b.flushers[numFlushers-1] = b.flushers[numFlushers-1], b.flushers[i]
```

Comment:
nit: Any reason we're swapping both positions? Seems it would be sufficient to just move the last entry into index `i` and then shorten the length of the slice.

Reply:
Sure sounds good.